11/changelog.yaml

```diff
@@ -4,9 +4,13 @@ releases:
   11.0.0a1:
     changes:
       removed_features:
-      - The ``inspur.sm`` collection was considered unmaintained and removed
-        from Ansible 11 (https://forum.ansible.com/t/2854).
-        Users can still install this collection with ``ansible-galaxy collection install inspur.sm``.
-      - The ``netapp.storagegrid`` collection was considered unmaintained and removed
-        from Ansible 11 (https://forum.ansible.com/t/2811).
-        Users can still install this collection with ``ansible-galaxy collection install netapp.storagegrid``.
+        - The ``inspur.sm`` collection was considered unmaintained and removed
+          from Ansible 11 (https://forum.ansible.com/t/2854).
+          Users can still install this collection with ``ansible-galaxy collection install inspur.sm``.
+        - The ``netapp.storagegrid`` collection was considered unmaintained and removed
+          from Ansible 11 (https://forum.ansible.com/t/2811).
+          Users can still install this collection with ``ansible-galaxy collection install netapp.storagegrid``.
+        - The ``frr.frr` has been removed because it does not support ansible-core 2.18
+          (https://forum.ansible.com/t/6243).
+        - The ``openvswitch.openvswitch` collection has been removed because it does not support ansible-core 2.18
+          (https://forum.ansible.com/t/6245).
```

Comment:
Can you remove the inserted spaces, so that the next release won't rewrite the first two entries again?

Reply:
Like [this](https://github.com/ansible-community/ansible-build-data/pull/437/commits/33a32a0d1fb86c42657df646376ef125d98a4941)?